README.md

```diff
@@ -49,12 +49,12 @@ A couple of useful references about virtual environments if you've never used th
 * [Virtual environments](https://docs.python-guide.org/dev/virtualenvs/)
 * [Creating virtual environments](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
 
-Install support for virtual environments with Python 3.x if you don't have it yet:
+Install support for virtual environments with Python 3.8.0 if you don't have it yet:
```

Comment:
I think we need to re-phrase this comment in conjunction with the modified command below